### DISCUSSION TITLE
Allow users to "Follow" updates for specific mod loaders

### DISCUSSION THREAD

**Discussion by ChristopherHaws** [modrinth/.github#85] [Feature requests]
### Is your feature request related to a problem? Please describe.

I use the Fabric mod loader exclusively. When forge version of a mod I follow are updated, I have to manually go through my notifications and manually dismiss those changes. Often I don't see that it was the forge version that updated and accidentally download it to my mods folder.

### Describe the solution you'd like

When I follow a mod, I would like to have the ability to pick which mod loader I want to follow. It would be extra awesome if I could follow only the "latest" Minecraft version!

### Describe alternatives you've considered

Alternatively having the ability to filter the notifications list by mod loader could help, but in reality I don't want the extra notifications.

**Comment by kosma** [comment]
I would also suggest following certain versions - I am not interested in a version I don't play.

**Reply by EssieFir** [reply]
Perhaps it could work similarly to YouTube, when you click follow, a button appears next to it which would allow you to customize the notifications you receive.
